delivery_driver_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -15,8 +15,19 @@ class StockPickingBatch(models.Model):
         store=True,
         domain="[('is_company', '=', False)]",
     )
+    user_id = fields.Many2one(
+        compute="_compute_user_id",
+        store=True,
+    )
 
     @api.depends("picking_ids.driver_id")
     def _compute_driver_ids(self):
         for batch in self:
             batch.driver_ids = batch.picking_ids.mapped("driver_id")
+
+    @api.depends("driver_ids")
+    def _compute_user_id(self):
```

Comment:
I'm not sure at all this is correct as this modifies the user_id values for all flows even without drivers.

Reply:
@rousseldenis do you know other module that computes this? If driver is not set or this module is not installed this don't impact